data/random-teams.ts

```diff
@@ -968,18 +975,20 @@ export class RandomTeams {
 			return {cull: !hasType['Poison'] && counter.Status >= 2};
 		case 'earthquake':
 			const doublesCull = hasMove['earthpower'] || hasMove['highhorsepower'];
-			const movePoolCull = movePool.includes('bodypress') && movePool.includes('shellsmash');
+			// Turtonator wants Body Press when it doesn't have Shell Smash
+			const turtonatorCull2 = movePool.includes('bodypress') && movePool.includes('shellsmash');
```

Comment:
First off, please name this `turtonatorCull` and not `turtonatorCull2`. It shouldn't conflict with the other one because it's in a parallel scope. Secondly, should these Turtonator-specific culls be gated on `species.id`? I don't think that Earthquake is better than Body Press generally; you could gate it on def > atk?